### PR TITLE
Added go_template for readiness & liveness probes (initialDelaySeconds)

### PR DESCRIPTION
**Pull request by kvitals**
we have had an issue with an 'initialDelaySeconds' on readiness probe - and predefined 5 seconds wasn't enough. So, we need this change to increase the value for this option. Kindly hope you'll find it useful ;) thanks

### CODE REVIEW

File: charts/victoria-metrics-cluster/values.yaml [hunk 1]
```diff
@@ -49,6 +49,19 @@ vmselect:
   extraLabels: {}
   env: []
 
+  # Readiness & Liveness probes
+  probe:
+    readiness:
+      initialDelaySeconds: 5
+      periodSeconds: 15
+      timeoutSeconds: 5
```
Comment (tenmozes) [inline charts/victoria-metrics-cluster/values.yaml:57]: this param can confuse as it is not been used in template but defined

File: charts/victoria-metrics-cluster/values.yaml [hunk 2]
```diff
@@ -49,6 +49,19 @@ vmselect:
   extraLabels: {}
   env: []
 
+  # Readiness & Liveness probes
+  probe:
```
Comment (tenmozes) [inline charts/victoria-metrics-cluster/values.yaml:53]: could you please use all params from values file in template or remove them

btw what about vminsert and storage
do you think, can this be helpful for other components?

**Review by tenmozes — CHANGES_REQUESTED** [review]

File: charts/victoria-metrics-cluster/values.yaml (hunk 1 above)
Comment (kvitals) [reply charts/victoria-metrics-cluster/values.yaml:57]: understood, already fixed it)

**Comment by kvitals** [comment]
hi @tenmozes 
Could you please check and approve?)

**Comment by tenmozes** [comment]
Thank you, i will test and approve, eta 1d

**Pull request merged**